README.md

```diff
@@ -251,8 +288,8 @@ Given `parseFooterTags: true`:
   "subject": "My first commit",
   "body": "Lorem ipsum dolor sit amet",
   "footer": {
-    "Closes": "#1",
-    "Foo": "bar"
+    "closes": "#1",
+    "foo": "bar"
```

Comment:
Yes, that wasn't updated on the change to multiple tags, the README.md should read:
```
    "Closes": "#1",
    "closes": "#1",
    "Floses": "#1",
    "floses": "#1",
```